### DISCUSSION TITLE
[Question] Does Lightroom store data about my web app?

### DISCUSSION THREAD

**Discussion by mattveraldi** [GoogleChrome/lighthouse#12896] [Q&A]
Hi, I would like to use Lighthouse to run tests against a web application that I'm developing for a client.
I would like to know If this tool (accessed by the developer console on Google Chrome or Edge) stores any information about the website I'm running those tests against.
Also I would like to know if it uses any Google or third party API to do such tests.

Thank you in advance.

**Comment by connorjclark** [comment]
> I would like to know If this tool (accessed by the developer console on Google Chrome or Edge) stores any information about the website I'm running those tests against.

Nope.

> Also I would like to know if it uses any Google or third party API to do such tests.

It all runs locally in DevTools and in the CLI. For the CLI, if you opt in we send crash reports to Sentry, which includes a URL.
